.github/workflows/backup-kubernetes-databases-inhouse.yml

```
@@ -0,0 +1,168 @@
+name: Backup kubernetes postgres databases.
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'Define the type of machine to run the job on'
+        type: string
+        required: false
+        default: '["self-hosted", "Linux", "X64", "bratislava"]'
+      backup:
+        description: 'Two types of backups postgress or shared-wordpress'
+        type: string
+        required: false
+        default: 'postgres'
+      cluster:
+        description: 'Kubernetes cluster name'
+        required: true
+        type: string
+      namespace:
+        description: 'Namespace where should be build app deployed'
+        required: true
+        type: string
+      url:
+        description: 'Kubernetes cluster url'
+        default: 'https://tkg.dev.bratislava.sk'
+        required: true
+        type: string
+    secrets:
+      service-account:
+        description: 'Kubernetes service account'
+        required: true
+      minio-endpoint:
+        description: 'Minio endpoint url'
+        required: true
+      minio-access:
+        description: 'Minio access key'
+        required: true
+      minio-secret:
+        description: 'Minio secret key'
+        required: true
+      minio-bucket:
+        description: 'Minio bucket name'
+        required: true
+      aws-key:
+        description: 'AWS key id'
+        required: true
+      aws-secret:
+        description: 'AWS secret access key'
+        required: true
+      aws-bucket:
+        description: 'AWS bucket name'
+        required: true
+
+jobs:
+  backup:
+    name: Backup databases
+    runs-on: ${{fromJSON(inputs.runs-on)}}
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Pipelines Version
+        run: |
+          echo "Pipelines version: 2.1.0"
+
+      - name: Directory check
+        run: pwd
+
+      - name: Kubectl tool installer
+        uses: Azure/setup-kubectl@v4.0.0
+
+      - name: Kubernetes set service account token
+        run: kubectl config set-credentials default --token=${{ secrets.service-account }}
+
+      - name: Kubernetes set server with certificate account token
+        run: kubectl config set-cluster ${{ inputs.cluster }}  --insecure-skip-tls-verify --server=${{ inputs.url }}
+
+      - name: Kubernetes set context cluster
+        run: kubectl config set-context ${{ inputs.cluster }} --cluster=${{ inputs.cluster }} --user=default
+
+      - name: Kubernetes use context
+        run: kubectl config use-context ${{ inputs.cluster }}
+      - name: Run database backup
```

All of the steps are separated by the new line, so this should be too
```suggestion
      
      - name: Run database backup
```